tests/system.py

```diff
@@ -736,6 +736,66 @@ def test_load_table_from_dataframe_w_automatic_schema(self):
         )
         self.assertEqual(table.num_rows, 3)
 
+    @unittest.skipIf(
+        pandas is None or pandas.__version__ < "1.0.1",
+        "Only `pandas version >=1.0.1` is supported",
+    )
+    @unittest.skipIf(pyarrow is None, "Requires `pyarrow`")
+    def test_load_table_from_dataframe_w_none_value(self):
```

Comment:
The nullable `Int64` data type is the real reason for the test. There are plenty of existing tests & samples with loading None values `object` dtype.